cloud_infrastructure_access/dedicated-aws-private-cluster.adoc

```diff
@@ -0,0 +1,16 @@
+[id="dedicated-aws-private-cluster"]
+= Configuring A Private Cluster
+include::modules/common-attributes.adoc[]
+:context: dedicated-private-cluster
+toc::[]
+
+An {product-title} cluster can be made private so that internal applications may be hosted inside a corporate network. In addition, private clusters can be configured to have internal only API endpoints for increased security.
```

Comment:
s/may/can

> ! may v
Use only to indicate permission. Otherwise, use “can” or “might.”

s/internal only/only internal